src/contrasts.jl

```diff
@@ -103,7 +103,7 @@ abstract type AbstractContrasts end
 # Contrasts + Levels (usually from data) = ContrastsMatrix
 struct ContrastsMatrix{C <: AbstractContrasts, M <: AbstractMatrix, T, U}
     matrix::M
-    termnames::Vector{U}
+    termnames::Vector{U} # XXX this is somewhat of a misnomer, this should be coefnames...
```

Comment:
You could rename the field but define a deprecated `getproperty` method to keep allowing `obj.termnames`.

Reply:
done